samples/cpp/speech_sample/speech_sample.hpp

```diff
@@ -175,8 +179,8 @@ DEFINE_int32(qb, 16, quantization_bits_message);
 /// @brief Scale factor for quantization
 DEFINE_string(sf, "", scale_factor_message);
 
-/// @brief Batch size (default 1)
-DEFINE_int32(bs, 1, batch_size_message);
+/// @brief Batch size (default 0)
+DEFINE_int32(bs, 0, batch_size_message);
```

Comment:
I propose don't change default value, but check whether command line attribute is specified. See `isFlagSetInCommandLine` from benchmark_app as example